Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs

```diff
@@ -476,6 +476,90 @@ public IEnumerator ManipulationHandlerOneHandMoveFar()
 
         }
 
+
+        /// <summary>
+        /// This tests the one hand near rotation and applying different rotation constraints to the object.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator ManipulationHandlerOneHandRotateWithConstraint()
```

Comment:
It would also be nice to test two hand rotation as well, unless you have other high priority bugs to get to for GA (it's a nice to have).

Reply:
I had a look , but the test for 2 hand rotation will be very different from this one - so i'll make a separate PR for that - i'll add it to the manipulation handler test task. Fixing the last GA bug first